person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
-  reader_attr :id
-  accessor_attr :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] As suggested by the previous reviewer, please set the `:rentals` to attribute accessor